hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/schemas/InitialModServiceTokenSchema.java

```diff
@@ -341,6 +342,12 @@ public void migrate(@NonNull final MigrationContext ctx) {
                                         aliasesState
                                                 .get()
                                                 .put(new ProtoBytes(toAcct.alias()), toAcct.accountIdOrThrow());
+                                        if (toAcct.alias().toByteArray().length > 20) {
```

Comment:
I'd like to see the PR comment "During BBM, add alias-to-id mappings for both the in-state alias (which could be a "key alias") and any implied EVM address" as a code comment here.

Reply:
Sure, I'll make sure to add that in my next PR. We've got a clean run on CI so I'm going to merge this one if that's alright